### DISCUSSION TITLE
Looking for a ready to go MSFS2024 TP page which works fine together  v1.6.0 - TP v4 for MSFS 2024

### DISCUSSION THREAD

**Discussion by SOFTWAREKOCH** [mpaperno/MSFSTouchPortalPlugin#71] [General]
Hi,

i installed the new v1.6.0 - TP v4 for MSFS 2024 and now i am looking for a ready to go example for MSFS2024 TP page, which covers not a specific aircraft only. The idea behind is, to take that one and adjust to my ideas

Thx for any suggestions in advance.

BTW : Thx to Max for doing the Upgrade of the Plugin for MSFS2024.

**Comment by mpaperno** [comment]
Hi Dietmar,

Out of curiosity, what kind of controls/displays are you looking for?  Anything specifically about FS 2024?

I know you've seen/installed the "Starter Page" example, which is probably as generic as anything I've seen out there.  It all seems to work in FS24, at least for the basic C172 and such.... but if not then maybe it needs an update or new version.  

It's hard to make a really generic page for all models (even of same class, eg. fixed wing single) because of model differences. 

I think that's one reason we see a lot of model-specific pages at https://flightsim.to/others/touch-portal-pages/new-updates/ for example, not much generic stuff.  Hopefully we'll get some updates from folks once FS24 is more stable (at least to confirm compatibility).  I wouldn't hesitate trying any of those with FS24, though of course some may have issues. And lots of interesting examples to copy from. (And BTW ignore the "deprecated" notice on Flak's excellent pages, they work fine now in TP v4.)

Cheers,
-Max

**Reply by SOFTWAREKOCH** [reply]
Hi Max,

i am looking for the ASOBO DA62, ASOBO C172.
In general GA Aircrafts

Brgd sDietmar
